scripts/fuzz_opt.py

```diff
@@ -618,7 +618,7 @@ def test_one(random_input, opts, allow_autoreduce):
                 # gets commands from the handler, for a given set of optimizations. this is all the commands
                 # needed to run the testing that that handler wants to do.
                 def get_commands(opts):
-                    return testcase_handler.get_commands(wasm='a.wasm', opts=opts + FUZZ_OPTS + FEATURE_OPTS, random_seed=random_seed)
+                    return testcase_handler.get_commands(wasm='a.wasm', opts=FUZZ_OPTS + opts + FEATURE_OPTS, random_seed=random_seed)
```

Comment:
`FUZZ_OPTS` has to come before `opts` because `FUZZ_OPTS` might contain a JS legalization pass that cannot operate on Stack IR.